<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SphericalHarmonicsEditor</class>
 <widget class="QWidget" name="SphericalHarmonicsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spherical Harmonics</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>56</width>
         <height>37</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxM0">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 10%);</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxM1">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 10%);</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0)</string>
            </property>
            <property name="text">
             <string>Coefficient 1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxM5">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 10%);</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_8">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0)</string>
            </property>
            <property name="text">
             <string>Power 1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0)</string>
            </property>
            <property name="text">
             <string>Coefficient 2</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxM7">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 10%);</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxM2">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 10%);</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0)</string>
            </property>
            <property name="text">
             <string>Power 4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0)</string>
            </property>
            <property name="text">
             <string>Coefficient 3</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_7">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0)</string>
            </property>
            <property name="text">
             <string>Power 2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxM4">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 10%);</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0)</string>
            </property>
            <property name="text">
             <string>Coefficient 4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0)</string>
            </property>
            <property name="text">
             <string>Power 3</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxM6">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 10%);</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxM3">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 10%);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
